src/System.Text.Json/tests/Serialization/Null.ReadTests.cs

```diff
@@ -29,6 +29,25 @@ public static void RootObjectIsNull()
             }
         }
 
+        [Fact]
+        public static void RootArrayIsNull()
+        {
+            {
+                int[] obj = JsonSerializer.Parse<int[]>("null");
+                Assert.Null(obj);
+            }
```

Comment:
I'm not used to see scope style on code base but I could  wrong @danmosemsft

Reply:
I'm not either, but I don't think we're against it either - particularly in test code if it makes things clearer.

In this case though arguably it's a "code smell" suggesting there should be 3 separate test methods.

Reply:
I'm not a huge fan of it myself, but it looked like an existing pattern in the test classes for short, repetitive tests like these.

Reply:
Yes it's only like this in tests and is there to avoid multiple tests and prevent these sections from accidently colliding on variable names. Sometimes it's used to verify a baseline condition before testing the primary case.